.evergreen/init.sh

```diff
@@ -190,3 +190,70 @@ _init_sh_evg_dir="$(dirname "${_init_sh_this_file}")"
 # this one, and will therefore remain as native paths
 EVG_DIR="$(native_path "${_init_sh_evg_dir}")"
 LIBMONGOCRYPT_DIR="$(dirname "${EVG_DIR}")"
+
+is_true() {
+    declare var="$1"
+    declare val="${!var-}"  # Default is '' empty
+    case "$val" in
+    1|true|TRUE|True|yes|YES|Yes|on|ON|On)
+        return 0;;
+    0|false|FALSE|False|no|NO|No|off|OFF|Off|"")
+        return 1;;
```

Comment:
I was going to suggest simplifying these comparison by using `declare -l val=...` to convert string to lowercase before comparison, but it seems like Bash on macos-1014 doesn't support it... 😕

Reply:
Yeah, I was unable to make this "pretty"...